shared/zodSchemas.ts

```diff
@@ -356,3 +353,62 @@ export const CursorPaginationDatetimeSchema = CursorPaginationSchema.extend({
 export type CursorPaginationDatetimeParams = z.infer<
   typeof CursorPaginationDatetimeSchema
 >;
+
+///////////////////////////////
+// INSTITUTION SCHEMAS
```

Comment:
Why is there a separate schema for each field of institution, and different names for the schema fields than in the institution model ("institutionName" vs just "name")?

I suggest maybe something like this for now:
```
export const InstitutionSchema = z.object({
  id: z.string().uuid(),
  createdAt: z.coerce.date(),
  updatedAt: z.coerce.date(),
  name: z.string().min(3, { message: "Institution name must at least 3 characters " }),
  domain: z.string().min(3, { message: "Institution domain must at least 3 characters " }),
});
export const createInstitutionSchema = InstitutionSchema .omit({
  id: true,
  createdAt: true,
  updatedAt: true,
});


```